performance-evaluation/setup-performance-evaluation.py

```diff
@@ -0,0 +1,384 @@
+			show_to = worksheet_destiny.rows if i + 1 == len(worksheet_talents) else list(filter(lambda each: each.value.startswith(worksheet_identifier + str(i + 2)), cells_with_titles))[0].row - 1
+			worksheet_destiny.show_rows(show_from, show_to)
+			print('Mostrando talento: ' + title)
+
+	print('')
+
+# This function is provisional and will be removed once the 'Insufficient tokens for quota' error is fixed
+def provisional_find_for_worksheet(worksheet, title):
+	matching_dictionary = {
+		'Universales': ['B2', 'B12', 'B23', 'B35', 'B45', 'B56', 'B67', 'B78', 'B88', 'B99'],
+		'Administración y Finanzas': ['B2', 'B12', 'B22'],
+		'Business Dev': ['B2', 'B12', 'B22', 'B32', 'B42', 'B52'],
+		'Calidad': ['B2', 'B12', 'B22', 'B32', 'B42'],
+		'Desarrollo': ['B2', 'B11', 'B19', 'B27', 'B35'],
+		'Diseño': ['B2', 'B10', 'B19', 'B28', 'B38'],
+		'QA': ['B2', 'B12', 'B23', 'B31', 'B41'],
+		'Referentes Técnicos': ['B2', 'B9'],
+		'Líderes': ['B2', 'B11', 'B21', 'B31', 'B41', 'B51', 'B61'],
+		'Marketing': ['B2', 'B11', 'B21', 'B31', 'B41'],
+		'Scrum Masters': ['B2', 'B12', 'B23', 'B33', 'B43'],
+		'People Care': ['B2', 'B13', 'B24', 'B35', 'B44'],
+		'Team Managers': ['B2', 'B12']
+	}
+
+	for key, value in matching_dictionary.items():
+		if title == key:
+			return list(map(lambda each: worksheet.cell(each), value))
+
+# Copy answers to destiny_sheet by reading them from answers_role_sheet in answers_role_row
+def copy_answers_role(destiny_sheet, answers_role_sheet, answers_role_row):
+	# Worksheet in index 0 to copy the answers 'Respuestas de formulario 1'
+	answers_worksheet_array = answers_role_sheet.worksheets()
+	answers_worksheet_index = next(index for index, each in enumerate(answers_worksheet_array) if each.title.startswith('Respuestas de formulario 1'))
+	answers_worksheet = answers_role_sheet.worksheets()[answers_worksheet_index]
+
+	# Worksheet in index 1 to paste the answers 'Satisfacción Laboral'
+	destiny_worksheet = list(filter(lambda each: each.index == 1, destiny_sheet.worksheets()))[0]
+
+	print('Realizando copia de respuestas del formulario de rol laboral a tab: ' + destiny_worksheet.title)
+	print('')
+
+	matching_dictionary = {
+		# Question: "¿En qué grado te sentís satisfecho/a con tu rol laboral actual?"
+		'D' + answers_role_row: ('B9', 'B10'),
+		# Question: "¿Por qué lo calificarías así?"
+		'F' + answers_role_row: ('B12', 'B13'),
+		# Question: ¿Qué otros aprendizajes te gustaría poder realizar en el transcurso de los próximos 6 meses?"
+		'G' + answers_role_row: ('B15', 'B16'),
+		# Question: ¿Qué otro rol y/o en qué otra área te gustaría poder trabajar a futuro dentro de Wolox?"
+		'H' + answers_role_row: ('B18', 'B19'),
+		# Question: "¿Qué aprendizajes considerás necesarios poder adquirir a fin de ejercer dicho rol? ¿En qué te gustaría capacitarte?"
+		'I' + answers_role_row: ('B21', 'B22')
+	}
+
+	for key, value in matching_dictionary.items():
+		destiny_question_cell = value[0]
+		destiny_answer_cell = value[1]
```

Comment:
If you want better naming these, you could make a dictionary:
`'I' + answers_role_row: {"question cell": 'B21', "answer cell": 'B22'}`
Or use namedtuple:
```
from collections import namedtuple
FormQuestion = namedtuple("FormQuestion", ["question_cel", "answer_cell"])
matching_dictionary = {
    'D' + answers_role_row: FormQuestion('B9', 'B10'),
    ...
}
for key, value in matching_dictionary.items():
    print('Copiando respuesta a: \'' + destiny_worksheet.cell(value.question_cell).value + '\'')
    ...
```